pkg/resourcemonitor/resourcemonitor.go

```diff
@@ -271,6 +273,20 @@ func (rm *resourceMonitor) updateNodeAllocatable() error {
 	return nil
 }
 
+func (rm *resourceMonitor) updateDevicesCapacity() {
```

Comment:
for consistency and logical ordering, we should probably update the capacity, then the allocatable. Not too big of a deal, though.

Reply:
We are relying on the allocatable value for updating the capacity since we do not have a trivial way to detect the capacity directly.

Reply:
ok, this is however a bit counter-intuitive so it deserves a code comment (perhaps bouncing back to the commit message/the place on which we spend time explaining why)

Reply:
Is the comment in lines 283-284 isn't enough?

Reply:
Contentwise - yes. But is buried very deep in the function, we should probably move it (better move than copy in this case IMO) upper in the stack. Perhaps in the call site?

Reply:
+1 on this.